tex/alg-NT/arithmetic.tex

```diff
@@ -0,0 +1,357 @@
+\chapter{Dirichlet's theorem on arithmetic progressions}
+
+\section{Introduction}
+
+In this chapter, we will sketch a proof of the following theorem:
+\begin{theorem}[Dirichlet's theorem on arithmetic progressions]
+	For every positive integers $a, n > 0$ such that $\gcd(a, n) = 1$, then there are infinitely
+	many primes coprime to $a$ modulo $n$.
+\end{theorem}
+
+For an even more concrete example:
+\begin{corollary}
+There are infinitely many primes $p$ that is congruent to $3$ modulo $4$.
+\end{corollary}
+
+You can see why this is called a theorem on arithmetic progressions:
+the example above is equivalent to saying that there are infinitely many primes in the arithmetic
+progression
+\[ 3, 7, 11, 15, 19, 23, \dots \]
+
+If you haven't noticed, the condition $\gcd(a, n) = 1$ is also very natural:
+look at the arithmetic progression $2, 6, 10, \dots$ where $a = 2$ and $n = 4$ are not coprime,
+and you will see that it obviously has only finitely many primes.
+
+(By the way, using Chebotarev density theorem which will be mentioned in the next section,
+you can even deduce something stronger: for example, a ``randomly chosen'' prime has probability
+$\frac{1}{2}$ to be congruent to $3$ modulo $4$. But here we will give a simple sketch of proof.)
+
+\section{The $L$-series}
+
+Actually, this chapter is just an excuse to introduce you to something called the $L$-series.
+Here is a quote from \cite{ref:neukirch} (emphasis mine):
+
+\begin{quote}
+One of the most astounding phenomena in number theory consists in the
+fact that a great number of deep arithmetic properties of a number field are
+hidden within a single analytic function, its \vocab{zeta function}. This function has
+a simple shape, but it is unwilling to yield its mysteries. Each time, however,
+that we succeed in stealing one of these well-guarded truths, we may expect to
+be rewarded by the revelation of some \emph{surprising} and \emph{significant} relationship.
+This is why zeta functions, as well as their generalizations, the $L$-series,
+have increasingly moved to the foreground of the arithmetic scene, and today
+are more than ever the focus of number-theoretic research.
+\end{quote}
+
+Surprisingly, you will see $L$-series in many other places as well --- an elliptic curve over $\QQ$
+has a $L$-series, and a modular form also have a $L$-series!\todo{really?
+And is there more examples?}
+
+So what is a $L$-series? In one sentence:\footnote{In my (limited) understanding, at the time of
+writing this.}
```

Comment:
Very unclear for the reader that the footnote is not written by Evan. May be a good place to put your disclaimer.